cmd/http-check/main.go

```diff
@@ -66,7 +66,7 @@ func main() {
 
 	// if the passing count is empty, then default to 1
```

Comment:
```suggestion
	// if the passing count is empty, then default to 100%
```

Reply:
yes it is @integrii everything checked out with this code for me locally